src/System.Runtime.Serialization.Primitives/src/System.Runtime.Serialization.Primitives.csproj

```diff
@@ -56,6 +56,10 @@
   <ItemGroup>
     <None Include="project.json" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">
+    <!-- TODO: remove this P2P reference once new packages are produced and updated -->
```

Comment:
Seems like we should have a single issue tracking cleaning up P2P's rather than lots of them. We al;ready have #10688, #12531, ...  and I imagine one person can make a single pass after NS2.0 work settles down next month.